src/frontend/scss/generic/_type.scss

```diff
@@ -8,9 +8,17 @@ $r-fonts: () !default;
 }
 
 // Build text color helpers
-@each $name, $value in $palette {
-  .text-#{$name} {
-    color: $value !important;
+@each $name, $value in $colors {
```

Comment:
Ditto, but this will maybe need a bigger refactor, we could talk about it during the next daily meeting.